src/vs/editor/common/viewModel/viewModelImpl.ts

```diff
@@ -266,8 +266,12 @@ export class ViewModel extends Disposable implements IViewModel {
 				for (const change of changes) {
 					switch (change.changeType) {
 						case textModelEvents.RawContentChangedType.LinesInserted: {
+							let injectedText: textModelEvents.LineInjectedText[] | null = null;
+							if (change.injectedText) {
+								injectedText = change.injectedText.filter(element => (!element.ownerId || element.ownerId === this._editorId));
+							}
 							for (const line of change.detail) {
-								lineBreaksComputer.addRequest(line, null, null);
+								lineBreaksComputer.addRequest(line, injectedText, null);
```

Comment:
I think there is something wrong here. `ModelRawLinesInserted` is an event that describes `N` inserted lines. But the field `ModelRawLinesInserted.injectedText` is not grouped in any way per line, it seems to be that all injected text is used on all new lines.

Reply:
Fixed.